src/InstrumentationEngine.Api/InstrumentationEngine.idl

```diff
@@ -1551,4 +1551,83 @@ library MicrosoftInstrumentationEngine
         HRESULT JitComplete([in] IMethodJitInfo* pJitInfo);
     };
 
-}
+    [
+        object,
+        uuid(EF0B0C79-08E7-4C3A-A4C5-02A9C9CE8809),
+        pointer_default(unique)
+    ]
+    interface IInstrumentationMethodSetting : IUnknown
+    {
+        // Gets the name of the instrumentation method setting.
+        HRESULT GetName(
+            [out] BSTR* pbstrName
+            );
+
+        // Gets the value of the instrumentation method setting.
+        HRESULT GetValue(
+            [out] BSTR* pbstrValue
+            );
+    };
+
+    [
+        object,
+        uuid(9B03D87E-72F0-4D8E-A4B1-15BCD8227073),
+        pointer_default(unique)
+    ]
+    interface IEnumInstrumentationMethodSettings : IUnknown
+    {
+        HRESULT Next(
+            [in] ULONG celt,
```

Comment:
>celt [](start = 23, length = 4)

what does "celt" stand for? and "rgelt"?

Reply:
Oh, elt = element

---
In reply to: [381505728](https://github.com/microsoft/CLRInstrumentationEngine/pull/204#discussion_r381505728) [](ancestors = 381505728)